test/cctest/util.cc

```diff
@@ -68,6 +68,21 @@ TEST(UtilTest, StringEqualNoCase) {
   EXPECT_FALSE(StringEqualNoCase("equals", "equal"));
 }
 
+TEST(UtilTest, StringEqualNoCaseN) {
+  using node::StringEqualNoCaseN;
+  EXPECT_FALSE(StringEqualNoCaseN("a", "b", 1));
+  EXPECT_TRUE(StringEqualNoCaseN("", "", 0));
+  EXPECT_TRUE(StringEqualNoCaseN("equal", "equal", 5));
```

Comment:
Can you use `strlen("equal")` or `sizeof("equal") - 1` instead of a number literal?  It looks more obviously correct.